Feature/ds 252 - Updated documentation voice and tone, as well as added some uniformity. (#383)
gsoft-inc/wl-hopper: apps/docs/content/getting-started/react.mdx

status: alpha
---

Hopper UI is a Design System that provides a collection of [components](components/component-list), [tokens](/tokens/overview/introduction), and [icons](/icons/overview/introduction) to help you build applications.

> Below is the typical installation path, assuming you are using React 18. If you have particluar needs have a look at these alternatives:

Reviewer: ```suggestion
Below is the typical installation path, assuming you are using React 18. If you have particular needs, have a look at these alternatives:
```

- [Javascript installation](/getting-started/javascript)

> - [Individual packages installation](/getting-started/individual-packages)

Reviewer: Just "package" is the correct way of saying it.

);
```

### Styling

> Import Hopper styles in your project by importing the following CSS files:

Reviewer: ```suggestion
Add Hopper styles to your project by importing the following CSS files:
```

```css
@import "@hopper-ui/styled-system/index.css";
@import "@hopper-ui/components/index.css";
@import "@hopper-ui/icons/index.css";
```

> Import Hopper fonts in your project by importing the following CSS file:

Reviewer: I feel like import is used too much. maybe use this:
```suggestion
Add Hopper fonts to your project by importing the following CSS file:
```
